src/test/java/ru/training/at/hw2/ex1/TestEx1.java

```diff
@@ -0,0 +1,140 @@
+package ru.training.at.hw2.ex1;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import javax.xml.crypto.Data;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import ru.training.at.hw2.data.TestData;
+
+public class TestEx1 extends TestData {
```

Comment:
How is `TestData` class related to `TestEx1` class? 
I suppose that it could be used as `TestData. TEXTS_OF_LOGS` in the tests.
There is no reasons for extending one class from other class.

Reply:
Yes, I fix it